cfme/tests/infrastructure/test_providers.py

```diff
@@ -117,16 +118,12 @@ def test_ip_required_validation():
 
 
 @pytest.mark.tier(3)
-@pytest.mark.xfail(message='http://cfme-tests.readthedocs.org/guides/gotchas.html#'
-    'selenium-is-not-clicking-on-the-element-it-says-it-is')
-def test_name_max_character_validation():
+def test_name_max_character_validation(provider, request):
     """Test to validate max character for name field"""
-    prov = VMwareProvider(
-        name=fauxfactory.gen_alphanumeric(256),
-        hostname=fauxfactory.gen_alphanumeric(5),
-        ip_address='10.10.10.12')
-    prov.create()
-    prov.delete(cancel=False)
+    provider.name = fauxfactory.gen_alphanumeric(255)
```

Comment:
This will fail if a previous provider with the same IP already exists. It will also be run multiple times for each provider due to the ```provider``` fixture being used. Instead I believe the issue is that we just need to add "any" credentials as well.